docs/cadence/tutorial/05-non-fungible-tokens-2.mdx

```diff
@@ -334,6 +342,16 @@ the Kitty literally stores the hat in its own storage and effectively owns it.
 The hat belongs to the CryptoKitty that it is stored in,
 and the hat can be transferred separately or along with the CryptoKitty that owns it.
 
+This also brings up an interesting wrinkle in Cadence in regards to ownership.
+In other ledger-based languages, ownership is indicated by account addresses.
+In cadence, since resources can own other resources, does it always make sense to say
+that the account that owns the resource that owns other resources also owns the
+sub-resources? Or are they simply the property of the resource? Its hard to say,
+and you'll encounter more fascinating implications of ownership and interoperability
+as you get deeper into Cadence. 
```

Comment:
This paragraph leaves the reader hanging, hints at object-orientation without mentioning it directly and is  vague and possibly confusing. We should be making clear and definite statements. How about: 

```suggestion
The reason that Resources can contain other Resources stems from the fact that Cadence is an object-oriented language. This is notably different from ledger-based blockchains whose _ledger_ is typically a key/value store in a contract, where the address of the user is associated to an interesting piece of data, eg: the amount of tokens owned by that address. 

A further distinction from ledger-based blockchains is that in Cadence all storage occurs in the account rather than the contract. The runtime transparently serializes and deserializes Resources - and the full graph of sub-Resources they may contain - when using the move operator. Ownership is intuitive and Resources enable a powerful level of interoperability and composability which you'll learn as you get deeper into Cadence. 
```

Reply:
I get what @franklywatson is getting at - that this is an opportunity to go deeper on the concept, but also agree with @joshuahannan that it may be missed given this is a simple tutorial. The second paragraph would be especially hard for someone new to Cadence to follow.

I see the original version to be a surface layer introduction to the idea and the suggestion to be a layer deeper. Could we perhaps add the suggestion to another section of the docs and link to it from the original.

Another option might be to add the suggested text in a toggle, so the information is there but requires action so reduces initial cognitive load - something like:

___

This also brings up an interesting wrinkle in Cadence in regards to ownership. In other ledger-based languages, ownership is indicated by account addresses. In cadence, since resources can own other resources, does it always make sense to say that the account that owns the resource that owns other resources also owns the sub-resources? Or are they simply the property of the resource? Its hard to say, and you'll encounter more fascinating implications of ownership and interoperability as you get deeper into Cadence - check out [the composable resource tutorial for just such an example](./10-resources-compose.mdx). 

<details>
<summary>🔎 Deeper Understanding - Resources & Ownership</summary>

The reason that Resources can contain other Resources stems from the fact that Cadence is an object-oriented language. This is notably different from ledger-based blockchains whose _ledger_ is typically a key/value store in a contract, where the address of the user is associated to an interesting piece of data, eg: the amount of tokens owned by that address. 

```solidity
// Ledger-based ownership - represented in a central contract
mapping(address => uint) public balanceOf // Address to balance mapping
```

A further distinction from ledger-based blockchains is that in Cadence all storage occurs in the account rather than the contract. 

![Vault ownership represented in account](https://github.com/onflow/docs/assets/108043524/fed258aa-7c4e-4904-a161-fe35a8a4201f)

The runtime transparently serializes and deserializes Resources - and the full graph of sub-Resources they may contain - when using the move operator.

Ownership is intuitive and Resources enable a powerful level of interoperability and composability which you'll learn more about as you get deeper into Cadence. 
</details>

___

I think the added detail is valuable, but I worry that newcomers might feel overwhelmed and intimidated seeing this info on the same level as all else in an intro tutorial.